src/components/Listing.tsx

```diff
@@ -1,39 +1,35 @@
 import React from "react";
 import { ListItem } from "../types/ListItem";
 import { ListingItem } from "../components/ListingItem";
-import { ItemType } from "../types/ItemType";
 
 export type ListingProps = {
   items: ListItem[];
-  itemChanged?: (listItem: ListItem) => void;
-  //onChange: (event: any) => void;
-};
-export const Listing: React.FC<ListingProps & ItemType> = ({
-  title,
-  items,
-  itemChanged,
-}) => {
-  return (
-    <>
-      <h2>{`${title} items`}</h2>
-      <table>
-        <thead>
-          <tr>
-            <th>URL</th>
-            <th>Date</th>
-          </tr>
-        </thead>
-        <tbody>
-          {items.map((item) => (
-            <ListingItem
-              item={item}
-              key={item.id}
-              title={title}
-              itemChanged={itemChanged}
-            />
-          ))}
-        </tbody>
-      </table>
-    </>
-  );
+  title: "Previously read links" | "Links to read";
+  itemChanged: (listItem: ListItem) => void;
 };
+export class Listing extends React.Component<ListingProps> {
+  render(): JSX.Element {
+    return (
+      <div>
+        <h2>{`${this.props.title} (${this.props.items.length})`}</h2>
```

Comment:
I think the outer code block is redundant and we could have only 
`{this.props.title} (${this.props.items.length})`
but that's rather cosmetical imo

Reply:
Yep, not even the `$` was neccessary.